### PR TITLE
docs(new docs site): removes old sphinx doc site with gatsby

### PR DESCRIPTION
**Pull request by pkdotson**
### SUMMARY
This pr remove the old sphinx generated doc site with the refreshed gatsby docz site. 

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ x] Introduces new feature or API
- [x ] Removes existing feature or API

related: https://github.com/apache-superset/docs/issues/18

### CODE REVIEW

File: docs/src/pages/docs/installation/upgrading_superset.mdx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+```
+docker-compose up
+```
+
+### Updating Superset Manually
+
+To upgrade superset in a native installation, run the following commands:
+
+```python
```
Comment (ktmud) [inline docs/src/pages/docs/installation/upgrading_superset.mdx:36]: The language should probably be `bash` or `shell`

File: docs/src/pages/docs/installation/sql_templating.mdx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+CUSTOM_TEMPLATE_PROCESSORS = {
+    CustomPrestoTemplateProcessor.engine: CustomPrestoTemplateProcessor
+}
+```
+
+SQL Lab also includes a live query validation feature with pluggable backends. You can configure
+which validation implementation is used with which database engine by adding a block like the
+following to your configuration file:
+
+```
```
Comment (ktmud) [inline docs/src/pages/docs/installation/sql_templating.mdx:80]: Nit 

```suggestion
```python
```

File: docs/src/pages/docs/installation/email_reports.mdx [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+
+### Schedule Reports
+
+You can optionally allow your users to schedule queries directly in SQL Lab. This is done by addding
+extra metadata to saved queries, which are then picked up by an external scheduled (like
+[Apache Airflow](https://airflow.apache.org/)).
+
+To allow scheduled queries, add the following to your configuration file:
+
+```
```
Comment (ktmud) [inline docs/src/pages/docs/installation/email_reports.mdx:84]:
```suggestion
```python
```

File: docs/src/pages/docs/installation/email_reports.mdx [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+of recipients (all of them can receive a single mail, or separate mails). For audit purposes, all
+outgoing mails can have a mandatory BCC.
+
+In order get picked up you need to configure a celery worker and a celery beat (see section above
+“Celery Tasks”). Your celery configuration also needs an entry `email_reports.schedule_hourly` for
+`CELERYBEAT_SCHEDULE`.
+
+To send emails you need to configure SMTP settings in your `superset_config.py` configuration file.
+
+```
```
Comment (ktmud) [inline docs/src/pages/docs/installation/email_reports.mdx:39]:
```suggestion
```python
```

File: docs/src/pages/docs/installation/configuring.mdx [hunk 5]
```diff
@@ -0,0 +1,282 @@
...
+SQLAlchemy connectors.
+
+Prior to SIP-15 SQLAlchemy used inclusive endpoints however these may behave like exclusive for
+string columns (due to lexicographical ordering) if no formatting was defined and the column
+formatting did not conform to an ISO 8601 date-time (refer to the SIP for details).
+
+To remedy this rather than having to define the date/time format for every non-IS0 8601 date-time
+column, once can define a default column mapping on a per database level via the `extra` parameter:
+
+```
```
Comment (ktmud) [inline docs/src/pages/docs/installation/configuring.mdx:236]:
```suggestion
```python
```

File: docs/src/pages/docs/installation/configuring.mdx [hunk 6]
```diff
@@ -0,0 +1,282 @@
...
+            me = self.appbuilder.sm.oauth_remotes[provider].get('userDetails').data
+            logging.debug("user_data: {0}".format(me))
+            return { 'name' : me['name'], 'email' : me['email'], 'id' : me['user_name'], 'username' : me['user_name'], 'first_name':'', 'last_name':''}
+    ...
+```
+
+This file must be located at the same directory than `superset_config.py` with the name
+`custom_sso_security_manager.py`. Finally, add the following 2 lines to `superset_config.py`:
+
+```
```
Comment (ktmud) [inline docs/src/pages/docs/installation/configuring.mdx:189]:
```suggestion
```python
```

File: docs/src/pages/docs/installation/configuring.mdx [hunk 7]
```diff
@@ -0,0 +1,282 @@
...
+---
+
+## Configuring Superset
+
+### Configuration
+
+To configure your application, you need to create a file `superset_config.py` and add it to your
+`PYTHONPATH`. Here are some of the parameters you can set in that file:
+
+```
```
Comment (ktmud) [inline docs/src/pages/docs/installation/configuring.mdx:16]:
```suggestion
```python
```

**Review by ktmud — APPROVED** [review]

**Comment by ktmud** [comment]
LGTM with a couple of nits. Can't wait to see this go live!

**Comment by mistercrunch** [comment]
For context, I've been stacking commits to fix CI. 
* removing all the build and documentation around Sphinx
* added license headers, couldn't find a way to get headers that both rat (the ASF license checker) and mdx like, I added mdx to excludes to move pass that

Only thing left is the `RELEASING/` docker cleanup. I feel like removing the automation in that area for now until we figure out how to version and auto-build.

File: docs/src/components/layout.css [hunk 8]
```diff
@@ -35,3 +35,18 @@ h1 {
 a {
   color: #1985a0;
 }
+
+h3 {
+  font-size: 30px;
+  font-weight: bold;
+  font-style: italic;
```
Comment (mistercrunch) [inline docs/src/components/layout.css:42]: italic?

**Comment by mistercrunch** [comment]
TODO: https://github.com/apache-superset/docs/issues/18

**Review by rusackas — APPROVED** [review]
LGTM! It's come a long way!

**Comment by mistercrunch** [comment]
success

**Comment by ktmud** [comment]
Congratulations, team! Thanks so much for taking on this!

One small style nit: 



Can we fix the hover color in the navigation menu?

**Comment by ronna** [comment]
on https://superset.apache.org/resources Installation. The install locally with docker link is broken. It's pointing to https://superset.incubator.apache.org/installation.html#start-with-docker instead of https://superset.apache.org/docs/installation/installing-superset-using-docker-compose

**Pull request merged**